src/NuGet.Indexing/NuGetQuery.cs

```diff
@@ -194,6 +197,11 @@ private static void AuthorClause(BooleanQuery query, Analyzer analyzer, IEnumera
             query.Add(ConstructClauseQuery(analyzer, "Authors", values), occur);
         }
 
+        private static void PackageTypeClause(BooleanQuery query, Analyzer analyzer, IEnumerable<string> values, Occur occur)
+        {
+            query.Add(ConstructClauseQuery(analyzer, "PackageTypesIndex", values), occur);
```

Comment:
General question about this file...
Should we put the `"PackageTypesIndex"`, `"Authors"`, and similar query names into constants? Is there already a constant with these values?